# History

## 0.1.0 - 2020-06-11

* First release on PyPI.

## 0.1.1 - 2020-07-05

### Bugs Fixed

* Numpy error when synthesising data with unique identifiers. - [Issue #23](https://github.com/DataResponsibly/DataSynthesizer/issues/23) by @raids
